speechmatics_flow/cli.py

```diff
@@ -99,18 +99,23 @@ def get_conversation_config(
     :param args: Keyword arguments probably from the command line.
     :type args: Dict
 
-    :return: Settings for the ASR engine.
+    :return: Settings for the Flow engine.
     :rtype: models.ConversationConfig
     """
 
-    config: Dict[str, Any] = {}
+    config = {"template_id": "default"}
+    # First, get configuration from the config file if provided.
     if args.get("config_file"):
         with open(args["config_file"], encoding="utf-8") as config_file:
             config = json.load(config_file)
 
     if config.get("conversation_config"):
         config.update(config.pop("conversation_config"))
```

Comment:
```suggestion
    if config.get("conversation_config"):
        config=config["conversation_config"]
```
I am not sure I would support this, but If so, I would do it this way so we drop everything that is not `conversation_config`